src/CommunityToolkit/Xamarin.CommunityToolkit/Converters/VariableMultiConverter.shared.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+using System.Globalization;
+using System.Linq;
+using Xamarin.CommunityToolkit.Extensions;
+using Xamarin.CommunityToolkit.Helpers;
+using Xamarin.Forms;
+using Xamarin.Forms.Internals;
+
+namespace Xamarin.CommunityToolkit.Converters
+{
+	public class VariableMultiConverter : MultiValueConverterExtension, IMultiValueConverter
+	{
+		public MultiBindingCondition ConditionType { get; set; }
+
+		public int Count { get; set; }
+
+		public object Convert(object[] values, Type targetType, object parameter, CultureInfo culture)
+		{
+			if (values == null || !targetType.IsAssignableFrom(typeof(bool)))
+				return false;
+
+			var amount = 0;
+
+			foreach (var value in values)
```

Comment:
BTW, this code doesn't handle **None** value if I am not wrong.

Also, I have a question about **Count**. It means "ExactValue". Perhaps it would be handy to have the possibility to set "At least {Count}" or "Not greater than {Count}"

What do you think?

Sorry, if I am too picky :) But I like this idea very much